src/model/data.types.model.ts

```diff
@@ -0,0 +1,12 @@
+export interface Address {
+    addressLine1?: string
+    addressLine2?: string
+    town?: string
+    county?: string
+    postcode?: string
+}
+
+export enum userResponse {
```

Comment:
seeing as this can only be YES or No, would calling it yesNoResponse be better? Might make it easier to understand when looking at the interfaces that use it

Reply:
Done